conans/test/integration/build_id_test.py

```diff
@@ -196,6 +196,30 @@ def _check():
         self.assertNotIn("ID: 4024617540c4f240a6a5e8911b0de9ef38a11a72", client.user_io.out)
         self.assertIn("ID: 5a67a79dbc25fd0fa149a0eb7a20715189a0d988", client.user_io.out)
 
+        client.run('install -s build_type=Debug')
+        client.run("info")  # Re-uses debug from curdir
+        _check()
+        self.assertNotIn("ID: 4024617540c4f240a6a5e8911b0de9ef38a11a72", client.user_io.out)
+        self.assertIn("ID: 5a67a79dbc25fd0fa149a0eb7a20715189a0d988", client.user_io.out)
+
+        client.run('install -s build_type=Release --install-folder=MyInstall')
+        client.run("info --install-folder=MyInstall")  # Re-uses debug from MyInstall folder
+        _check()
+        self.assertIn("ID: 4024617540c4f240a6a5e8911b0de9ef38a11a72", client.user_io.out)
+        self.assertNotIn("ID: 5a67a79dbc25fd0fa149a0eb7a20715189a0d988", client.user_io.out)
+
+        client.run('install -s build_type=Debug --install-folder=MyInstall')
+        client.run("info --install-folder=MyInstall")  # Re-uses debug from MyInstall folder
+        _check()
+        self.assertNotIn("ID: 4024617540c4f240a6a5e8911b0de9ef38a11a72", client.user_io.out)
+        self.assertIn("ID: 5a67a79dbc25fd0fa149a0eb7a20715189a0d988", client.user_io.out)
+
+        # Both should raise
+        error = client.run("info --install-folder=MyInstall -s build_type=Release",
```

Comment:
I would expect tests for this functionality in test/command/info... it is a bit weird that they are mixed with ``build_id()`` tests